sbtc/src/lib.rs

```diff
@@ -46,4 +55,34 @@ pub const NUMS_X_COORDINATE: [u8; 32] = [
 /// sBTC deposit transactions, this public key is such that it does not
 /// have a known private key.
 pub static UNSPENDABLE_TAPROOT_KEY: LazyLock<XOnlyPublicKey> =
-    LazyLock::new(|| XOnlyPublicKey::from_slice(&NUMS_X_COORDINATE).unwrap());
+    LazyLock::new(|| XOnlyPublicKey::from_slice(&DERIVED_NUMS_X_COORDINATE).unwrap());
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use bitcoin::bip32::{ChainCode, ChildNumber, DerivationPath, Fingerprint, Xpub};
+    use secp256k1::{PublicKey, Secp256k1};
+    use std::str::FromStr;
+
+    /// This test proves, that the DERIVED_NUMS_X_COORDINATE is correct,
+    /// and is derived from the NUMS_X_COORDINATE using the derivation path "0/0".
+    #[test]
+    fn test_derivation() {
```

Comment:
One alternative is to define the `UNSPENDABLE_TAPROOT_KEY` using the same logic of this test.